pasta_eln/GUI/project.py

```diff
@@ -343,8 +344,11 @@ def modelChanged(self, item:QStandardItem) -> None:
       stackNew.append(docIDj)
     stackNew = [self.projID] + stackNew[::-1]  #add project id and reverse
     childNew = item.row()
-    if doc['-type'][0][0]=='x':
-      dirNameNew= createDirName(doc['-name'],doc['-type'][0],childNew) # determine path: do not create yet
+    if branchOld['path'] is not None and not branchOld['path'].startswith('http'):
+      if doc['-type'][0][0]=='x':
+        dirNameNew= createDirName(doc['-name'],doc['-type'][0],childNew) # determine path: do not create yet
```

Comment:
The comment and function name (createDirName) does not match, correct the comment?

Reply:
Thank you, did it